src/MusicStore/MusicStore.csproj

```diff
@@ -11,80 +11,81 @@
     
     <!-- This will prevent our build system from trying to package this project. -->
     <IsPackable>false</IsPackable>
+    
+    <!-- This will be set as an environment variable to pin the package version. -->
+    <AspNetVersion>$(JITBENCH_ASPNET_VERSION)</AspNetVersion>
```

Comment:
This is gross but necessary - we need to pin so that we use the same packages as the zip we downloaded.